libraries/helpers.rb

```diff
@@ -750,3 +753,46 @@ def check_ssh_target_checker_vpc_cidr_list(ssh_target_checker_script, expected_c
     TEST
   end
 end
+
+def copy_to_device(directory, device_name, fs_type, mount_options)
```

Comment:
i thought that during offline discussion we ruled out the idea of copying files on behalf of the user?

Reply:
Yes, the PR is in draft because at some point it did not reflect the last decisions we made. Nevertheless, I thought it would have been useful to have an initial rev of the approach.
Will fix in next rev.